package.json

```diff
@@ -9,7 +9,11 @@
 		"start": "bsb -make-world -w -ws _ ",
 		"storybook": "start-storybook -p 6006",
 		"test": "jest",
-		"tdd": "jest --watchAll"
+		"tdd": "jest --watchAll",
+		"precommit": "lint-staged"
+	},
+	"lint-staged": {
+		"*.{re,rei}": ["refmt --in-place", "git add"]
```

Comment:
The `git add` stages _all_ files, right? Isn't that quite error prone? What if you only staged a hunk before, won't this also pick up the remaining changes?

Reply:
Yeah, the lint-staged thing only works on the things that are staged. It takes whatever is staged, lints it, and then re-adds it. So also hunks are supported :)